msp/cost/master_org_cost_policy_currency/master_org_cost_policy_currency.pt

```diff
@@ -0,0 +1,1026 @@
+name "Master Org Cost Policy with Currency Conversion"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy allows you to set up cross organization scheduled reports that will provide summaries of cloud cost across all organizations you have access to, delivered to any email addresses you specify. The policy will report the following:
+Chart of the previous 6 months of utilization based on [category](https://docs.rightscale.com/optima/reference/rightscale_dimensions.html#category).  
+Daily average cost across the last week and last month.  
+Total cost during previous full week (Monday-Sunday) and previous full month.  
+Total cost during current (incomplete) week and month.  
+We recommend running this policy on a weekly cadence and applying it to your master account.
+_Note 1: The last 3 days of data in the current week or month will contain incomplete data._  
+_Note 2: The account you apply the policy to is unimportant as Optima metrics are scoped to the Org._  
+_Note 3: Exchange rates are calculated at execution time using [https://exchangeratesapi.io/](https://exchangeratesapi.io/)._  
+See [README](https://github.com/rightscale/policy_templates/tree/master/msp/cost/master_org_cost_policy) for more details"
```

Comment:
The link to this readme is not accurate.

Reply:
@cdwilhelm resolved